doc/barclamp.md

```diff
@@ -1356,3 +1356,154 @@ place) you will have to add additional setup code to
 and/or
 [qa_crowbarsetup.sh](https://github.com/SUSE-Cloud/automation/blob/master/scripts/qa_crobarsetup.sh)
 to take care of your barclamp's needs.
+
+## High Availability (HA) Integration
+
+Using the [barbican barclamp](https://github.com/crowbar/crowbar-openstack/tree/master/chef/cookbooks/barbican)
+as an example. We will try to investigate what changes are required to make a barclamp
+HA compatible.
+
+[This commit](https://github.com/crowbar/crowbar-openstack/pull/512/files) is
+the first place we start looking. It was not entirely correct and needed
+[more commits](https://github.com/crowbar/crowbar-openstack/commits/master/chef/cookbooks/barbican)
+to correct. There are a lot of potential mistakes in getting HA working, maybe 
+whatever you are struggling with right now turned up in the Barbican barclamp as well.
+
+-  [default.rb](https://github.com/crowbar/crowbar-openstack/commits/master/chef/cookbooks/barbican/attributes/default.rb)
+
+   This file defines the defaults for all the values of the vars required by barclamp.
```

Comment:
thanks a lot for writing that. Explains a lot.  [Stackexhange](https://meta.stackexchange.com/a/7837) also had something interesting about the number of spaces to get indented code blocks

Done!